ifrs17/Import/Importers.ipynb

```diff
@@ -310,50 +314,82 @@
                 "\n    if(ApplicationMessage.HasErrors()) return null;",
-                "\n    ",
+                "\n",
                 "\n    var main = mainTab.Rows.First();",
-                "\n    var reportingNode = (string)main[nameof(ReportingNode)];",
-                "\n    var scenario = mainTab.Columns.Where(x => x.ColumnName == nameof(Scenario)).Count()>0? (string)main[nameof(Scenario)] : default(string);",
-                "\n   ",
+                "\n    var reportingNode = mainTab.Columns.Any(x => x.ColumnName == nameof(Args.ReportingNode)) ? (string)main[nameof(ReportingNode)] : default(string);",
+                "\n    var scenario = mainTab.Columns.Any(x => x.ColumnName == nameof(Args.Scenario)) ? (string)main[nameof(Scenario)] : default(string);",
+                "\n    var year = mainTab.Columns.Any(x => x.ColumnName == nameof(Args.Year)) ? (int)Convert.ChangeType(main[nameof(Args.Year)], typeof(int)) : default(int);",
+                "\n    var month = mainTab.Columns.Any(x => x.ColumnName == nameof(Args.Month)) ? (int)Convert.ChangeType(main[nameof(Args.Month)], typeof(int)) : default(int);",
+                "\n",
+                "\n    return new ImportArgs(reportingNode, year, month, default(Periodicity), scenario, default(string));",
+                "\n}"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "markdown",
+            "source": [
+                "A validation method which checks year and month. Log errors if any is missing. "
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "public static void ValidateArgsForPeriod(this ImportArgs args) {",
+                "\n    if(args.Year == default(int)) ApplicationMessage.Log(Error.YearInMainNotFound);",
+                "\n    if(args.Month == default(int)) ApplicationMessage.Log(Error.MonthInMainNotFound);",
+                "\n}"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "markdown",
+            "source": [
+                "## Get Args and create Partition",
+                "\n",
+                "\nThis is the main method to get Args which reference a specific data partition. All validations are triggered after parsing and if the partition is not already existing in the DataSource it is created. "
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "public async Task<ImportArgs> GetArgsAndCommitPartitionAsync<IPartition>(IDataSet dataSet)",
```

Comment:
the method name contains "Commit" whereas the markdown above and the section title contain "create". We should uniform this.